tshirts.cpp

```diff
@@ -14,9 +14,22 @@ char size(int cms) {
 }
 
 int main() {
+    // Test cases for size 'S'
     assert(size(37) == 'S');
+    assert(size(0)  == 'S');
+    assert(size(38) == 'S');
+    assert(size(-1) == 'S');

```

Comment:
When you get a shoulder size of `-1` (which is probably a mistake), do you want to really ship a t-shirt with size `S`?